src/main/java/org/folio/inventory/support/http/server/ForwardResponse.java

```diff
@@ -3,28 +3,33 @@
 import io.vertx.core.buffer.Buffer;
 import io.vertx.core.http.HttpHeaders;
 import io.vertx.core.http.HttpServerResponse;
+
+import org.apache.commons.lang3.StringUtils;
 import org.folio.inventory.support.http.client.Response;
 
 public class ForwardResponse {
   private ForwardResponse() { }
 
+  public static void forward(HttpServerResponse forwardTo, Response forwardFrom) {
+    forward(forwardTo, forwardFrom.getBody(), forwardFrom.getStatusCode(),
+      forwardFrom.getContentType());
+  }
+
   public static void forward(HttpServerResponse forwardTo,
-                             Response forwardFrom) {
+    String body, int statusCode, String contentType) {
 
-    forwardTo.setStatusCode(forwardFrom.getStatusCode());
+    forwardTo.setStatusCode(statusCode);
 
-    if(forwardFrom.hasBody()) {
-      Buffer buffer = Buffer.buffer(forwardFrom.getBody(), "UTF-8");
+    if (StringUtils.isNotBlank(body)) {
+      Buffer buffer = Buffer.buffer(body, "UTF-8");
 
-      forwardTo.putHeader(HttpHeaders.CONTENT_TYPE, forwardFrom.getContentType());
+      forwardTo.putHeader(HttpHeaders.CONTENT_TYPE, contentType);
       forwardTo.putHeader(HttpHeaders.CONTENT_LENGTH, Integer.toString(buffer.length()));
```

Comment:
Whilst these lines are changing, please could the header constants be statically imported.